### ISSUE TITLE
Titlebar appears

### ISSUE THREAD

**Issue by mastodon** [concerto/concerto-player#29]
I am running Ubuntu player built on your chroot_tasks.sh and I see a titlebar. Either blackbox has changed or Chromium is unable to enter true fullscreen. In either way I suspect that this change will at some point affect the player built with this script.

**Comment by augustf** [comment]
I'm not experiencing this issue under the normal Debian build with Chromium, but we'll track this issue for a bit to see if there's anything we can do.

**Comment by asquared** [comment]
I built the player several times last night and did not notice this.

**Comment by asquared** [comment]
The player should be built using Debian. Perhaps this issue is caused by differences in Ubuntu.

**Comment by mastodon** [comment]
Debian and Ubuntu have somewhat different blackbox versions, so I'm closing this one. Sorry for bothering and I hope that it's not an upstream issue.

For those adventurous who are also wokring from Ubuntu-based player and end up here: my solution to the titlebars problem:

I installed ratpoison window manager and wrote .ratpoisonrc file (goes to concerto user directory) containing:

    msgwait 1
    hsplit -2
    banish

The first line makes the message disappear slightly faster. Then there is a problem with Chromium image tooltip sometimes appearing even if the mouse is not moved: for a workaround I split the screen so that the rightmost side is 2 pixels smaller and banish mouse to the bottom-right corner: if you use TVs you won't miss those pixels much.

Then I replaced the line on blackbox with ratpoison in .xinitrc
